design/iterations.go

```diff
@@ -24,6 +24,9 @@ var iterationAttributes = a.Type("IterationAttributes", func() {
 	a.Attribute("name", d.String, "The iteration name", func() {
 		a.Example("Sprint #24")
 	})
+	a.Attribute("description", d.String, "Description of the iteration ", func() {
+		a.Example("this is the description of iteration")
```

Comment:
The example could be little more realistic. How about this? `Sprint #24 focusing on UI changes`